gumroad/scripts/gumblocks.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * GumBlocks
+ *
+ * Widgets and blocks for the Gutenberg editor.
+ *
+ *
+ *
+ *
+*/
+
+(function( blocks, element ) {
+  const el = element.createElement;
+  const {TextControl} = wp.components;
+
+  import { TextControl } from '@wordpress/components';
+  import { withState } from '@wordpress/compose';
+
+  blocks.registerBlockType( 'gumroad/gumblocks', {
+    title: 'Gumroad Embed',
```

Comment:
think we should support embed, overlay, and link. see this pr: #25 

will also effect how we do shortcodes. will elaborate more on that issue too.